clients/cloud/python/README.md

```diff
@@ -17,6 +17,24 @@ sasl.username=<api-key-id>
 sasl.password=<secret-access-key>
 ```
 
+* Properly configured trust store on client system
+
+In order for the producer to verify the certificate of the brokers make sure the trust store is set up correctly on your system. 
+
+** CentOS
+
+```bash
+$ sudo yum reinstall ca-certificates
+```
+
+Add the following property to the `Producer` and `AdminClient` constructors in `producer.py`:
+
+```
+ssl.ca.location='/etc/pki/tls/cert.pem'
```

Comment:
Shouldn't this be in the config dict, i.e.,:
`"ssl.ca.location": "/etc/pki/tls/cert.pem"`

Reply:
Are you sure this is the file that should be used and not /etc/ssl/certs/ca-bundle.crt ?

Reply:
On my CentOS machine `/etc/ssl/certs/ca-bundle.crt` and `/etc/pki/tls/cert.pem` are both symlinked to `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`. These files are all managed by the `ca-certificates` package. Shall I change the property to `/etc/ssl/certs/ca-bundle.crt`?

Reply:
Fixed https://github.com/confluentinc/examples/pull/144/commits/ad8efa1f74a377c1d596cb6c0323830a22e42a54